mailing_list/templates/mailing_list/admin_index.html

```diff
@@ -2,75 +2,62 @@
 
 {% load static %}
 
-
 {% block body %}
-<body class="lti-tool" role="application" ng-controller="MailingListController as ml">
-    <header>
-        <h1>{% block breadcrumbs %}Course Emailer{% endblock breadcrumbs %}</h1>
-    </header>
-    <div class="loading-indicator" ng-show="ml.isLoading">
-        <img src="{% static 'mailing_list/images/ajax-loader-large.gif' %}"/>
+<body class="lti-tool"
+      role="application"
+      ng-controller="MailingListController as ml">
+  <header>
+    <h1>
+      {% block breadcrumbs %}Course Mailing List{% endblock breadcrumbs %}
+    </h1>
+  </header>
+  <div class="loading-indicator" ng-show="ml.isLoading">
+    <img src="{% static 'images/ajax-loader-large.gif' %}"/>
+  </div>
+  <main class="ng-hide" ng-show="ml.loaded">
+    <div id="course-sections" ng-show="ml.hasPrimarySections()">
+      <div ng-repeat="list in ml.primarySectionLists">
+        {% verbatim %}
+        <p class="caption-primary" ng-show="$first">
+          <a href="{{ ml.listMembersUrl(list) }}">
+            View course members' email addresses</a>
+          ({{ list.members_count }} members)
```

Comment:
Sounds like this should be inside the <a> tag above as well, via the UI/UX team.